modules/front-end/src/app/core/components/env-drawer/env-drawer.component.html

```diff
@@ -21,7 +21,7 @@
       <nz-form-item>
         <nz-form-label nzRequired i18n="@@common.name">Name</nz-form-label>
         <nz-form-control nzErrorTip="Environment name is mandatory!" i18n-nzErrorTip="@@org.project.envNameMandatory">
-          <input type="text" nz-input (ngModelChange)="nameChange($event)" formControlName="name" placeholder="Name"
+          <input type="text" nz-input formControlName="name" placeholder="Name"
                  i18n-placeholder="@@common.name"/>
         </nz-form-control>
```

Comment:
Formatting suggestion: put `i18n-placeholder` and the `placeholder` attribute on the same line.